nixos/modules/services/misc/flaresolverr.nix

```diff
@@ -0,0 +1,50 @@
+{
+  config,
+  pkgs,
+  lib,
+  ...
+}:
+
+with lib;
+
+let
+  cfg = config.services.flaresolverr;
+in
+{
+  options = {
+    services.flaresolverr = {
+      enable = mkEnableOption "FlareSolverr, a proxy server to bypass Cloudflare protection";
+
+      package = mkPackageOption pkgs "flaresolverr" { };
+
+      openFirewall = mkOption {
+        type = types.bool;
+        default = false;
+        description = "Open ports in the firewall for the Flaresolverr web interface.";
+      };
+    };
+  };
+
+  config = mkIf cfg.enable {
+    systemd.services.flaresolverr = {
+      description = "Flaresolverr";
+      after = [ "network.target" ];
+      wantedBy = [ "multi-user.target" ];
+
+      environment = {
+        HOME = "/run/flaresolverr";
+      };
+
+      serviceConfig = {
```

Comment:
i [found](https://github.com/FlareSolverr/FlareSolverr/blob/805a34c9d621cf39c04ca799a5df26ac85dcfd56/flaresolverr.service) a `systemd` service by upstream which included `TimeoutStopSec=30` and `RestartSec=5`

given what this software does i think those probably make sense to include here too